### DISCUSSION TITLE
use this smart contract on others

### DISCUSSION THREAD

**Discussion by gunomnifluencer** [smartcontractkit/full-blockchain-solidity-course-py#369] [Q&A] [answered]
Hello,

In this tutorial Patrick used etherium as example currency, 

Can used this tutorial to make smart contract on other currency such bit coin or etc?

please answer

Thanks

**Comment by cromewar** [comment, marked as answer]
Hello @gunomnifluencer the answer is no, but with nuances:
There are two types of blockchains: type 1 like bitcoin, type 2 like ethereum. A blockchain type two allows you to run code, and a operating system inside of it (EVM on ethereum), if you want to develop smart contracts you must use a blockchain type two (bitcoin is not one of them) like ethereum and solana, you also are going to pay fees on the governance currency token of that blockchain.

With that said, you can 100% use bitcoin or any other token you want on your smart contract but as I said you must develop and deploy it using a blockchain type 2.
